app/models/patient.rb

```diff
@@ -138,6 +143,103 @@ class Patient < ApplicationRecord
       .where('public_health_action = ?', 'None')
   }
 
+  # All individuals currently being monitored if true, all individuals otherwise
+  scope :with_active_monitoring, lambda { |active_monitoring|
+    where('monitoring = ?', true) if active_monitoring
```

Comment:
Could you do this in a different way, like instead using a `.where.not`: https://thoughtbot.com/blog/activerecords-wherenot